cmd/maya-exporter/app/collector/cstorcollector.go

```diff
@@ -179,6 +179,11 @@ func (c *Cstor) parser(stats v1.VolumeStats) VolumeStats {
 	volStats.totalReadBytes, _ = stats.TotalReadBytes.Float64()
 	volStats.totalWriteBytes, _ = stats.TotalWriteBytes.Float64()
 	volStats.sectorSize, _ = stats.SectorSize.Float64()
+	volStats.totalReadTime, _ = stats.TotalReadTime.Float64()
+	volStats.totalWriteTime, _ = stats.TotalWriteTime.Float64()
+	volStats.totalReadBlockCount, _ = stats.TotalReadBlockCount.Float64()
+	volStats.totalWriteBlockCount, _ = stats.TotalWriteBlockCount.Float64()
```

Comment:
Can we have a IO latency as well at the lower level, instead of calculating it in control plane ?
Or do we have any challenges there ?

Reply:
what do you refering to `at the lower level` graffana ?
as per the design perspective we will not calculate anything here, it will just redirect whatever it got.
cc: @AmitKumarDas @avishnu